data/software-tools/flake8.json

```diff
@@ -0,0 +1,26 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "rs": "https://w3id.org/everse/rs#"
+  },
+  "@id": "https://w3id.org/everse/tools/flake8",
+  "@type": "schema:SoftwareApplication",
+  "schema:name": "flake8",
+  "schema:description": "flake8 is a Python linting tool that checks code for style and programming errors. Integrates multiple tools (pyflakes, pycodestyle, etc.) and supports plugin extensions.",
+  "schema:url": "https://flake8.pycqa.org/",
+  "schema:isAccessibleForFree": true,
+  "rs:hasQualityDimension": {
+    "@id": "rs:Flexibility",
```

Comment:
are linters really improving _Flexibility_?
My first thought would be _Maintainability_